### ISSUE TITLE
tunnelservers: ipv6 config

### ISSUE THREAD

**Issue by pmelange** [freifunk-berlin/puppet#75]
In considering setting up ipv6 for the tunnels, we need to first gather some information about the current ipv6 settings.

# a.tunnel (vm02.roellig-it.de)
* No IPv6
* DHCP Client
* community: Sven1.0 (contacted.  The VM-Host has a /64 and he has not yet assigned a part of that to the VM).

# b.tunnel (freifunk-gw01.in-berlin.de)
* Yes IPv6
* Static Addresses
* contact: admin@in-berlin.de

# c.tunnel (vpn03d)
* No IPv6
* Static ipv4 Address
* community: Freifunk Rheinland

# d.tunnel (vpn03f)
* Yes IPv6
* Static Addresses
* community: Freifunk Hamburg

# e.tunnel (vpn03g)
* Yes IPv6
* Static Addresses
* community: Freifunk NordWest?

# f.tunnel (vpn03h)
* No IPv6
* DHCP Client
* hosted by netcup

**Comment by booo** [comment]
> The above-mentioned goals of RFC 3177 can easily be met by giving
home users a default assignment of less than /48, such as a /56.

https://tools.ietf.org/html/rfc6177

How many end sites aka nodes do we have per server?

**Comment by pmelange** [comment]
according to http://monitor.berlin.freifunk.net/host.php?h=tunnel-clients&p=exec there are 53 clients per server

**Comment by booo** [comment]
I suggest we plan for growth and assume 128 clients per sever.

We have 6 servers. In the worst case we have 6 * 128 on one server. That's 768 clients on one server. We round to 1024.

So we need 1024 * /56 for each server to be on the save side. So we need 2**10 subnets. 56 - 10 is /46.

**Comment by pmelange** [comment]
https://tools.ietf.org/html/rfc7421 section 3 describes using prefixes longer than /64

**Comment by booo** [comment]
Freifunk Rheinland can assign us a /48 but we need a RIPE handle to get the IPs.

**Comment by booo** [comment]
@pmelange Do we have a maximum number of clients configured for the tunneldigger setup?

**Comment by pmelange** [comment]
> @pmelange Do we have a maximum number of clients configured for the tunneldigger setup?

Yes we do.  Just under 4000.  https://github.com/freifunk-berlin/puppet-communitytunnel/blob/4c34b78c85d2f58efc1fb8ae060a799cec8f2c25/manifests/init.pp#L8
